clannetwork/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "clannetwork",
+    "assets": [
+      {
+        "description": "The native token of Clan Network",
+        "denom_units": [
+          {
+            "denom": "uclan",
+            "exponent": 0
+          },
+          {
+            "denom": "clan",
+            "exponent": 6
+          }
+        ],
+        "base": "uclan",
+        "name": "Clan Network",
+        "display": "clan",
+        "symbol": "CLAN",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/clannetwork/images/clan.png",
```

Comment:
Although these URIs look good, the images being uploaded look more like a chain logo rather than a token logo. Please upload token logo images instead